### PR TITLE
Remove"Email/Password should not be empty" message

### PR DESCRIPTION
**Pull request by monkaushik10**
Remove"Email/Password should not be empty" message when focus moves to that field.

### CODE REVIEW

File: lib/src/10_auth/auth_screen.dart [hunk 1]
```diff
@@ -157,27 +184,37 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
...
     return Column(children: <Widget>[
-      TextField(
-        autofocus: true,
-        controller: emailController,
-        decoration: InputDecoration(
-          labelText: resolvePhrase(FauiPhrases.EmailTextField,"Email"),
+      Focus(
+        child: TextField(
+           autofocus: true,
+           controller: emailController,
```
Comment (polina-c) [inline lib/src/10_auth/auth_screen.dart]: Can we wrap "Focus(...)" into function?

File: lib/src/10_auth/auth_screen.dart [hunk 2]
```diff
@@ -157,27 +184,37 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
...
+           decoration: InputDecoration(
+            labelText: resolvePhrase(FauiPhrases.EmailTextField, "EMail"),
```
Comment (polina-c) [inline lib/src/10_auth/auth_screen.dart]: Can you make it 'Email' not 'EMail' everywhere?

**Review by polina-c — CHANGES_REQUESTED** [review]

File: lib/src/10_auth/auth_screen.dart (hunk 1 above)
Comment (monkaushik10) [reply lib/src/10_auth/auth_screen.dart]: Done. The function is callled " _buildTextBox"

File: lib/src/10_auth/auth_screen.dart (hunk 2 above)
Comment (monkaushik10) [reply lib/src/10_auth/auth_screen.dart]: Done

File: lib/src/10_auth/auth_screen.dart [hunk 3]
```diff
@@ -128,9 +128,58 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
...
+          autofocus: (field == 'Email') ? true : false,
+          controller: controllerName,
+          decoration: InputDecoration(
+            labelText: resolvePhrase(phrase, field),
+          ),
+          onSubmitted: (s) {
+            submit();
+          },
+        ),
+        onFocusChange: (hasFocus) {
```
Comment (polina-c) [inline lib/src/10_auth/auth_screen.dart:150]: you are doing the same thing in all these cases. Why you are checking condition at all? Can you just set _error to ""?

File: lib/src/10_auth/auth_screen.dart [hunk 4]
```diff
@@ -128,9 +128,58 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
...
+    } else if (field == 'Password') {
+      phrase = FauiPhrases.PasswordTextField;
+    }
+    return Focus(
+        child: TextField(
+          autofocus: (field == 'Email') ? true : false,
+          controller: controllerName,
+          decoration: InputDecoration(
+            labelText: resolvePhrase(phrase, field),
+          ),
```
Comment (polina-c) [inline lib/src/10_auth/auth_screen.dart:145]: Better pass already resolved phrase here.

File: lib/src/10_auth/auth_screen.dart [hunk 5]
```diff
@@ -128,9 +128,58 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
...
+      String screen, Future<Widget> Function() submit) {
+    FauiPhrases phrase;
+    if (field == 'Email') {
+      phrase = FauiPhrases.EmailTextField;
+    } else if (field == 'Password') {
+      phrase = FauiPhrases.PasswordTextField;
+    }
+    return Focus(
+        child: TextField(
+          autofocus: (field == 'Email') ? true : false,
```
Comment (polina-c) [inline lib/src/10_auth/auth_screen.dart:141]: What will happen if autofocus will be true for both? False for both?

**Review by polina-c — CHANGES_REQUESTED** [review]

**Pull request closed without merging**